demisto_sdk/commands/validate/validators/BC_validators/BC107_is_valid_toversion_on_modified.py

```diff
@@ -14,30 +16,111 @@
 
 class IsValidToversionOnModifiedValidator(BaseValidator[ContentTypes]):
     error_code = "BC107"
-    description = (
-        "Check that the toversion property was not changed on existing Content files."
-    )
+    description = "Check that the toversion property was not changed on existing Content files. unless there's a new file with the same id and continues from_version."
     rationale = "Changing the `toversion` field for a content item can break backward compatibility."
-    error_message = "Changing the maximal supported version field `toversion` is not allowed. Please undo, or request a force merge."
+    error_message = "Changing the maximal supported version field `toversion` is not allowed. unless you're adding new content_item with the same id {0} and their from/to version fulfills the following:\n{1}"
     related_field = "toversion"
-    expected_git_statuses = [GitStatuses.MODIFIED, GitStatuses.RENAMED]
+    expected_git_statuses = [
+        GitStatuses.MODIFIED,
+        GitStatuses.RENAMED,
+        GitStatuses.ADDED,
+    ]
+    valid_cases = (
+        "The old item `fromversion` field should be less than the new item `fromversion` field\n"
+        "The old item `toversion` field should be less than the new item `fromversion` field\n"
```

Comment:
combined with BA118, the first check is redundant

Reply:
also, where is the check discussed that "the newest item has no toversion"?

Reply:
it's not the same in BA118, BA118 if for the same file "toversion" and "fromversion" fields.
my first condition is for the old and new file.
 "the newest item has no toversion" is a case that's not related to my changes in specific, aka we were supposed to already have this validation  (new items has no toversion).
 im discussing it with the relevant side now. we need a new BA validation.

Reply:
if we know old.from < old.to (BA118)
and old.to < new.from (the 2nd condition above)
then old.from < new.from